crmsh/ui_resource.py

```diff
@@ -338,7 +338,7 @@ def do_promote(self, context, rsc):
         if not xmlutil.RscState().is_ms_or_promotable_clone(rsc):
             logger.error("%s is not a promotable resource", rsc)
             return False
-        role = "Promoted" if config.core.OCF_1_1_SUPPORT else "Master"
+        role = utils.handle_role_for_ocf_1_1("Master")
```

Comment:
We could also use the constant RSC_ROLE_PROMOTED_LEGACY here.

Reply:
Changed, 
Thanks!